smart-contract-extensions/hardhat.config.js

```diff
@@ -18,9 +23,31 @@ const settings = {
   },
 }
 
+const networks = {}
+
+// add mainnet fork -- if API key is present
+if (process.env.RUN_MAINNET_FORK) {
+  // eslint-disable-next-line no-console
+  console.log('Running a mainnet fork...')
+  const alchemyAPIKey = process.env.ALCHEMY_API_KEY
+  if (!alchemyAPIKey) {
+    throw new Error('Missing Alchemy API Key, couldnt run a mainnet fork')
+  }
+  const alchemyURL = `https://eth-mainnet.alchemyapi.io/v2/${alchemyAPIKey}`
```

Comment:
Let's just use our RPC https://rpc.unlock-protocol.com so we don't have to mess with API keys :)